reporter/src/main/kotlin/model/EvaluatedModelMapper.kt

```diff
@@ -162,6 +163,15 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
         }
     }
 
+    private fun createScopes() {
+        input.ortResult.getProjects().flatMap { project -> project.scopes.map { it.name } }.forEach { scope ->
```

Comment:
Nit: How about expanding the formatting a bit like

        input.ortResult.getProjects().flatMap { project ->
            project.scopes.map { it.name }
        }.forEach { scope ->
            scopes[scope] = EvaluatedScope(
                name = scope,
                excludes = scopeExcludes.addIfRequired(input.ortResult.getExcludes().findScopeExcludes(scope))
            )
        }

Reply:
Done.